packages/jest-reporters/src/CoverageReporter.ts

```diff
@@ -456,7 +450,7 @@ export default class CoverageReporter extends BaseReporter {
                   originalSource: fileTransform.originalCode,
                   source: fileTransform.code,
                   sourceMap: {
-                    sourcemap: {file: res.url, ...sourcemapContent},
+                    sourcemap: {file: res.url, ...sourcemapContent} as any,
```

Comment:
<img width="539" alt="image" src="https://user-images.githubusercontent.com/1404810/164022791-89b4999b-eb77-440b-8455-abf1af85e4c6.png">

https://github.com/istanbuljs/v8-to-istanbul/blob/1f357fa2e994385b56ba505ca199eda46b233a89/index.d.ts#L14
https://github.com/mozilla/source-map/blob/58819f09018d56ef84dc41ba9c93f554e0645169/source-map.d.ts#L17
vs
https://github.com/jridgewell/trace-mapping/blob/d4ead19a18c805bd3af657f71ea72c48809f61db/src/types.ts#L7